package/osx/make-package

```diff
@@ -313,20 +311,8 @@ if [ "${build_package}" = "1" ]; then
    cd "${BUILD_DIR_X86_64}"
    "${CMAKE}" --build . --target install -- ${MAKEFLAGS}
 
-   if [ "${rstudio_target}" = "Electron" ]; then
-      OLDPATH="${PATH}"
-      NODEPATH="$(dirname "${NODE}")"
-      YARNPATH="$(dirname "${YARN}")"
-      PATH="${NODEPATH}:${YARNPATH}:${OLDPATH}"
-
-      # Build and package the Electron app.
-      cd "${ELECTRON_SRC_DIR}"
-      ./scripts/update-json-version.sh "$RSTUDIO_VERSION_FULL"
-      yarn make
-      cd "${PKG_DIR}"
-      PATH="${OLDPATH}"
+   if [ "${build_dmg}" = "1" ]; then
 
-   elif [ "${build_dmg}" = "1" ]; then
       # move to target directory
       cd "${INSTALL_DIR}"
```

Comment:
Two approaches to consider here:

- Copy the session components into the Electron build tree, and use Electron forge to produce the final package build;
- Copy the session components into an Electron-specific sub-directory, merge the Electron build together with those session bits, and produce the built DMG ourselves. (Basically, mimicing the Electron packaging process, but still using our existing CMake tools to produce a DMG)

Reply:
I think 2nd approach is better due to not having to deal with some issues that electron could bring when copying before building the binary...

Reply:
Sounds good! This is what @gtritchie and I were leaning towards, as well.